mnefun/_mnefun.py

```diff
@@ -949,7 +949,6 @@ def run_sss_locally(p, subjects, run_indices):
             raw.fix_mag_coil_types()
             raw = filter_chpi(raw)
 
-            assert isinstance(p.trans_to, string_types)
             if p.trans_to is 'median':
```

Comment:
I mean I missed it in review...

Don't worry about putting the assert back. I was initially thinking that
it's bad to keep allowing options that only work with the Python algorithm,
but I think it's okay.